api/notd/store/retriever.py

```diff
@@ -104,6 +104,13 @@ async def get_most_traded_token(self, startDate: datetime.datetime, endDate: dat
         row = result.first()
         return Token(registryAddress=row[TokenTransfersTable.c.registryAddress], tokenId=row[TokenTransfersTable.c.tokenId])
 
+    async def list_tokens_by_collection(self, address: str, connection: Optional[DatabaseConnection] = None) -> List:
```

Comment:
this query looks wierd, can you post here the sql it outputs pls

Reply:
SELECT DISTINCT tbl_token_transfers.registry_address, tbl_token_transfers.token_id 
FROM tbl_token_transfers 
WHERE tbl_token_transfers.registry_address = :registryAddress_1

Reply:
SELECT DISTINCT tbl_token_transfers.registry_address, tbl_token_transfers.token_id 
FROM tbl_token_transfers 
WHERE tbl_token_transfers.registry_address = :registryAddress_1
[('0x7509615baf2f9d2eE3274BDa6CC1FC444e0488e8', '2796'), ('0x7509615baf2f9d2eE3274BDa6CC1FC444e0488e8', '3995'), ('0x7509615baf2f9d2eE3274BDa6CC1FC444e0488e8', '3996'), ('0x7509615baf2f9d2eE3274BDa6CC1FC444e0488e8', '3997'), ('0x7509615baf2f9d2eE3274BDa6CC1FC444e0488e8', '5470'), ('0x7509615baf2f9d2eE3274BDa6CC1FC444e0488e8', '5604'), ('0x7509615baf2f9d2eE3274BDa6CC1FC444e0488e8', '6717') ...

Reply:
nice, thanks